layouts/product-landing.html

```diff
@@ -26,13 +26,13 @@ <h1 class="mb-3 font-mktg">
             </a>
 
             {% if page.introLinks.reference %}
-              <a href="{{ page.introLinks.reference }}" class="ml-3 btn-mktg btn-outline-mktg btn-large f4 mt-3">
+              <a href="{{ page.introLinks.reference }}" class="btn-mktg btn-outline-mktg btn-large f4 mt-3">
```

Comment:
Rather than just removing `ml-3` to remove the left margin here, let's replace it with `mr-3` to add the margin to the right side instead. This is work well on mobile while also looking as expected on larger displays.

```suggestion
              <a href="{{ page.introLinks.reference }}" class="btn-mktg btn-outline-mktg btn-large f4 mt-3 mr-3">
```

Reply:
Does it make sense at the moment?
For the 3rd button is it? because there is no content there

Reply:
cc @JamesMGreene